ibm/service/power/ibm_pi_constants.go

```diff
@@ -157,6 +157,7 @@ const (
 	Attr_WorkspaceName         = "pi_workspace_name"
 	Attr_WorkspaceStatus       = "pi_workspace_status"
 	Attr_WorkspaceType         = "pi_workspace_type"
+	Attr_powerEdgeRouterStatus = "pi_powerEdgeRouterStatus"
```

Comment:
Please stylize the name the same way as the other variables.
"Attr_PowerEdgeRouterStatus"